src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,27 @@
 import React from 'react';
+import { Comment } from '../../types/Comment';
+import { CommentInfo } from '../CommentInfo';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
-);
+interface Props {
+  comments: Comment[],
+}
+
+export const CommentList: React.FC<Props> = (props) => {
+  if (props.comments.length === 0) {
+    return (
+      <>
+        <hr />
+
+        <b data-cy="NoCommentsMessage">No comments yet</b>
+      </>
+    );
+  }
+
+  return (
+    <div className="CommentList">
+      {props.comments.map(comment => (
+        <CommentInfo {...comment} />
```

Comment:
```suggestion
        <CommentInfo {comment} />
```
add there <li> tag with key